src/utils/fit.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import { type PositionT } from '../flowTypes'
+
+export const isPositionedToTop = (position: PositionT): boolean => {
+  const [positionMarker] = position.split(' ')
+
+  return positionMarker === 'top'
+}
+
+export const isPositionedToBottom = (position: PositionT): boolean => {
+  const [positionMarker] = position.split(' ')
+
+  return positionMarker === 'bottom'
+}
+
+export const isPositionedToRight = (position: PositionT): boolean => {
+  const [positionMarker] = position.split(' ')
```

Comment:
Don't get this. Shouldn't it be `const positionMarker = position.split(' ')[1]`?

Reply:
I suspect this is a bug, but there is no test covering this.

Reply:
You might be right about that :D